src/cdef.rs

```diff
@@ -128,10 +127,9 @@ fn constrain(diff: i32, threshold: i32, damping: i32) -> i32 {
 // of the 2-pixel padding around the block, not the block itself.
 // The destination is unpadded.
 unsafe fn cdef_filter_block<T: Pixel>(
-  dst: *mut T, dstride: isize, input: *const T, istride: isize, pri_strength: i32,
+  dst: *mut T, dstride: isize, input: *const u16, istride: isize, pri_strength: i32,
```

Comment:
Are we then roll back to Monty's original code that always used 16bit for input? I remember you have rescued him when all those Pixel Type changes (by rom1v) has perplexed him with haunting rebasing the other day.

Reply:
The input code should be much as it was before the Pixel type changes. On the output side, we now additionally have 8bit via the Pixel type.